coralproject/talk
# Email Confirm Redirect URI

## What does this PR do?

Fixes a bug when changing your email address, the redirect URI is wrong.

## How do I test this PR?

1. Create an account.
2. Go to settings under "My Profile"
3. Change your email address.
4. Click the confirmation email for the change.
5. Click to confirm the change.
6. Notice the correct redirect!
